zstacklib/zstacklib/utils/sanlock.py

```
@@ -488,7 +488,14 @@ def calc_id_renewal_fail_seconds(io_timeout):
 
 
 def calc_host_dead_seconds(io_timeout):
-    return 8 * int(io_timeout) + get_watchdog_fire_timeout()
+    return calc_id_renewal_fail_seconds(io_timeout) + get_watchdog_fire_timeout()
+
+
+def monotime():
+    try:
+        return int(linux.monotime())
+    except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)